style/style.js

```diff
@@ -14,3 +14,10 @@ export const mediaquery = {
   desktopToBigScreen: 'min-width: 1200px',
   bigScreenBreakpoint: '1200px',
 }
+
+export const mediaqueryGrid = {
```

Comment:
DevNote: Adds breakpoints the grid is using. It probably makes sense to sync the grid with the already defined breakpoints. We just need to figure out which work better for us.